.globalconfig

```diff
@@ -1400,7 +1400,7 @@ dotnet_diagnostic.SA1514.severity = none
 dotnet_diagnostic.SA1515.severity = none
 
 # SA1516: Elements should be separated by blank line
-dotnet_diagnostic.SA1516.severity = none
+dotnet_diagnostic.SA1516.severity = warning
```

Comment:
@xtqqczze Did you you check that rule forcing works?

Reply:
It works, see for example #14136:

`/home/vsts/work/1/s/src/powershell/Program.cs(187,1): error SA1507: Code should not contain multiple blank lines in a row [/home/vsts/work/1/s/src/powershell-unix/powershell-unix.csproj]`

_Originally posted by @xtqqczze in https://github.com/PowerShell/PowerShell/issues/14136#issuecomment-730040038_